src/Css.elm

```diff
@@ -3375,3 +3386,50 @@ inset =
 outset : Value { provides | outset : Supported }
 outset =
     Value "outset"
+
+-- TEXT TRANSFORM --
+
+{-| Sets [`text-transform`](https://css-tricks.com/almanac/properties/t/text-transform/).
+
+    textTransform capitalize
+    textTransform uppercase
+
+-}
+textTransform :
+    Value
+        { capitalize : Supported
+        , uppercase : Supported
+        , lowercase : Supported
+        , full-width : Supported
+        , none : Supported
+        , initial : Supported
+        , inherit : Supported
+        , unset : Supported
+        }
+    -> Style
+textTransform (Value str) =
+    AppendProperty ("text-transform:" ++ str)
+
+{-| A `capitalize` [text transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).
+-}
+capitalize : Value { provides | capitalize : Supported }
```

Comment:
Can `capitalize` (and the other values - `uppercase`, `lowercase`, and `fullWidth`) get code examples like the one `textTransform` has?

Even something as basic as showing `textTransform capitalize` can show people a valid way to use this value.